ethdb/database.go

```diff
@@ -178,16 +188,22 @@ func (db *LDBDatabase) Meter(prefix string) {
 //      2   |        523 |    1000.37159 |       7.26059 |      66.86342 |      66.77884
 //      3   |        570 |    1113.18458 |       0.00000 |       0.00000 |       0.00000
 //
+// This is how the write delay look like (currently):
+// DelayN:5 Delay:406.604657ms
+//
 // This is how the iostats look like (currently):
 // Read(MB):3895.04860 Write(MB):3654.64712
 func (db *LDBDatabase) meter(refresh time.Duration) {
 	// Create the counters to store current and previous compaction values
 	compactions := make([][]float64, 2)
 	for i := 0; i < 2; i++ {
-		compactions[i] = make([]float64, 3)
+		compactions[i] = make([]float64, 5)
 	}
 	// Create storage for iostats.
 	var iostats [2]float64
+	// Track the time of the last write delay warning log print
+	var lastWriteDelay time.Time
+	var lastWriteDelayN time.Time
```

Comment:
Please leave an empty line after this (also after the `iostats` declaration, should have been fixed in the original code).